ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline API
+
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource
+Resource            ../../../Resources/Page/Operators/OpenShiftPipelines.resource
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Suite Setup         General Suite Setup
+Suite Teardown      General Suite Teardown
+
+
+*** Variables ***
+${PROJECT_USER3}=    dspa-project-user3
+${PROJECT_USER4}=    dspa-project-user4
+${S3_BUCKET}=    ods-ci-ds-pipelines
+
+
+*** Test Cases ***
+Verify Ods User Can Bind The Route Role
+    [Documentation]    Create A Data Science Projects for user3
+    ...         Create A Data Science Projects for user4
+    ...         Test with user3 can access the project for user4, it should fail because it doesn't have the permission
```

Comment:
hm should it be "test user3 can access the pipeline route created by user4" instead of project?

Reply:
Done